models/dex/solana/dex_solana_bot_trades.sql

```diff
@@ -0,0 +1,50 @@
+{{
+    config(
+        schema = 'dex_solana',
+        alias = 'bot_trades',
+        materialized = 'view',
+        post_hook='{{ expose_spells(\'["solana"]\',
+                                    "project",
+                                    "dex",
+                                    \'["hosuke"]\') }}'
```

Comment:
can we make it a habit to use the arg names in this `expose_spells` post hook macro, similar to one i linked to in merged PR today for balances?

we don't need to worry about applying to all existing spells (yet), but rather focus on making it a practice moving forward